src/routes/api/users.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import loginRoute from './users/login';
+import registerRoute from './users/register';
+import verifyRoute from './users/verify';
+import { resetPasswordRoute, changePasswordRoute } from './users/passwordReset';
+import bodyVerify from '../../middleware/bodyVerify';
+
+const router = Router();
+
+router.post('/users/login', bodyVerify(['email', 'password']), loginRoute);
+router.post('/users/register', bodyVerify(['name', 'email', 'password']), registerRoute);
+router.post('/users/verify', bodyVerify(['email', 'code']), verifyRoute);
+router.post('/users/resetPassword', bodyVerify(['email']), resetPasswordRoute);
+router.post('/users/changePassword', bodyVerify(['email', 'password', 'code']), changePasswordRoute);
```

Comment:
Since this router would be nested under `/api/users`, all you need here is `/changePassword` (and similarly `/login`, etc). With how things are currently setup, I believe these routes are accessible via `/api/users/users/changePassword` which I'm assuming isn't intended.